commit ed2af275251147815ca007177b4e3b3b5ea2f045 (HEAD -> master)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-09-13T16:50:35-04:00 (3 weeks ago)
    
    TRITON-1899 sdcnode builds for minimal-64-19.2.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 3d637ca..199f76d 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -1,4 +1,33 @@
 [
+    {
+        "// image": "minimal-64 19.2.0",
+        "image": "7f4d80b4-9d70-11e9-9388-6b41834cbeeb",
+        "version": "v8.16.1",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v8.x.patch",
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-64 19.2.0",
+        "image": "7f4d80b4-9d70-11e9-9388-6b41834cbeeb",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v8.16.1",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v8.x.patch",
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-64 19.1.0",
         "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
@@ -290,6 +319,37 @@
         "gcc": "/opt/local/gcc49/bin/gcc",
         "build_tag": "zone"
     },
+    {
+        "// image": "minimal-64 19.2.0",
+        "image": "7f4d80b4-9d70-11e9-9388-6b41834cbeeb",
+        "version": "v6.17.0",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v6.17+.patch",
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "npm_version": "v6.4.1",
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-64 19.2.0",
+        "image": "7f4d80b4-9d70-11e9-9388-6b41834cbeeb",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.17.0",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v6.17+.patch",
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "npm_version": "v6.4.1",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-64 19.1.0",
         "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
